ui/src/components/common/Grapher.js

```diff
@@ -202,6 +203,15 @@ class Grapher extends Component {
                         <i className="fa fa-close fa-1x close-btn" onClick={hideProps}/>
                         {this.state.selectedTask.taskType} ({this.state.selectedTask.status})
                     </h4>
+                    {this.state.selectedTask.taskType == 'SUB_WORKFLOW' &&
+                        <div>
+                            <p>
+                            <Link onClick={hideProps} to={'/workflow/id/' + this.state.selectedTask.outputData.subWorkflowId}>
```

Comment:
`subWorkflowId` has since moved to task object. This should instead be `this.state.selectedTask.subWorkflowId`

Reply:
Thank you for checking it out! I moved `subWorkflowId` to the task object.